nilearn/interfaces/fmriprep/tests/utils.py

```diff
@@ -4,24 +4,59 @@
 
 import pandas as pd
 
+from nilearn._utils.data_gen import _create_bids_filename
+from nilearn.interfaces.bids._utils import _bids_entities
 from nilearn.interfaces.fmriprep import load_confounds_utils
 
 img_file_patterns = {
     "ica_aroma":
-        "_space-MNI152NLin2009cAsym_desc-smoothAROMAnonaggr_bold.nii.gz",
-    "regular": "_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz",
-    "res": "_space-MNI152NLin2009cAsym_res-2_desc-preproc_bold.nii.gz",
-    "native": "_desc-preproc_bold.nii.gz",
-    "cifti": "_space-fsLR_den-91k_bold.dtseries.nii",
-    "den": "_space-fsLR_den-32k_desc-preproc_bold.nii.gz",
-    "part": (
-        "_part-mag_space-MNI152NLin2009cAsym_"
-        "desc-preproc_bold.nii.gz"
-    ),
+        {"suffix": "bold",
+         "entities": {"space": "MNI152NLin2009cAsym",
+                      "desc": "smoothAROMAnonaggr"},
+         "extension": "nii.gz"},
```

Comment:
I'm thinking that the extension could be ".nii.gz" by default, following BIDS conventions.

Reply:
wait do you mean that it should be `.nii.gz` instead of `nii.gz` like written here?
or that we should set a default at tall?

Reply:
No, sorry, I don't make a difference between .nii.gz and nii.gz. I just mean that this is recommended image format in BIDS.

Reply:
OK then I can see if I can simplify this a little bit.

Reply:
OK nii.gz will be set as default extension and bold as default suffix when creating the filename